google-cloud-error_reporting/lib/google/cloud/error_reporting.rb

```diff
@@ -0,0 +1,258 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google-cloud-error_reporting"
+require "google/cloud/error_reporting/project"
+require "google/cloud/error_reporting/middleware"
+require "stackdriver/core"
+
+module Google
+  module Cloud
+    ##
+    # # Stackdriver ErrorReporting
+    #
+    # Stackdriver Error Reporting counts, analyzes and aggregates the crashes in
+    # your running cloud services. The Stackdriver Error Reporting API provides:
+    # * [A simple endpoint to report errors from your application](
+    # #report-error).
```

Comment:
The same anchor syntax is [currently being used](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/ced1cb4de543ce06312f13ef816a370c18cab76d/google-cloud-logging/lib/google/cloud/logging.rb#L29-L34), and it seems to be working on the github.io docs.